### ISSUE TITLE
vgg16 network performance with arcface loss 

### ISSUE THREAD

**Issue by jetsmith** [auroua/InsightFace_TF#33]
hi, considering the speed, I want to use vgg16 trained on the dataset without alignment. i have trained the vgg16 model without fc layers on MSCeleb Aligned Facec Images(using top 2000 class), achieve the testing accuracy  86.2% on unaligned lfw onefold. Now i want to use arcface loss to impove the performance. and  choose the vggface2 as the training set (no alignment),  vgg  have no fc layers  followed by arcface loss and softmax, set the margin 0.3(mentioned on the issues), and the trained model will be tested on the unligned LFW onefold, the optimizer is the same as yours
question 1: have you tried the vgg16 model with  arcface loss, what's your accuracy? And my net's accuracy could reach more than 97% ?
question 2: are there any suggestions for me?
many thanks!
